web/src/app/users/linkToMobile/VerifyCodeFields.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react'
+import {
+  DialogContent,
+  DialogContentText,
+  Grid,
+  TextField,
+  makeStyles,
+} from '@material-ui/core'
+import gql from 'graphql-tag'
+import { useMutation } from 'react-apollo'
+
+const useStyles = makeStyles({
+  codeContainer: {
+    marginTop: '2em',
+  },
+  contentText: {
+    marginBottom: 0,
+  },
+  textField: {
+    textAlign: 'center',
+    fontSize: '1.5rem',
+  },
+})
+
+const mutation = gql`
+  mutation($input: VerifyAuthLinkInput!) {
+    verifyAuthLink(input: $input)
+  }
+`
+
+interface VerifyCodeFieldsProps {
+  authLinkID: string
+}
+
+export default function VerifyCodeFields(
+  props: VerifyCodeFieldsProps,
+): JSX.Element {
+  const classes = useStyles()
+  const [numOne, setNumOne] = useState('')
+  const [numTwo, setNumTwo] = useState('')
+  const [numThree, setNumThree] = useState('')
+  const [numFour, setNumFour] = useState('')
```

Comment:
Can we make this a little more generic? Take a prop like `length`, store the code as a single string in state.

Reply:
Will be resolved transitioning to verifying without re-typing